test/TestExpectations.json

```diff
@@ -2719,6 +2705,13 @@
     "expectations": ["FAIL"],
     "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
   },
+  {
+    "testIdPattern": "[network.spec] network Page.authenticate should allow disable authentication",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["firefox", "webDriverBiDi"],
+    "expectations": ["FAIL"],
+    "comment": "The implememtation does not expect 2 responseCompleted events"
```

Comment:
Could you clarify which implementation?

Reply:
Done, its Puppeteer.
Though the (proposed) Fetch Spec change don't feel easy to implement.